internal/sdks/sdk_instructions/python.md

```diff
@@ -0,0 +1,56 @@
+# Set up your application
+
+If you want to skip ahead, the final code is available in our [GitHub repository](https://github.com/launchdarkly/hello-python).
+
+
+1. Create a new directory:
+
+```bash
+mkdir hello-python && cd hello-python
+```
+
+2. Next, create a file called `requirements.txt` with the SDK dependency and install it:
+
+```bash
+echo "launchdarkly-server-sdk==9.3.1" >> requirements.txt && pip install -r requirements.txt
+```
+
+3. Create a file called `test.py` and add the following code:
+
+```python
+# Import the LaunchDarkly client.
+import ldclient
+from ldclient import Context
+from ldclient.config import Config
+
+# Create a helper function for rendering messages.
+def show_message(s):
+    print("*** {}".format(s))
```

Comment:
I _did_ change the string formatting from what's shown in the quickstart because our rendering was trying/failing to find the `%s` values and instead was showing `MISSING` values

before:
```python
print("*** %s" % s)
```
![image](https://github.com/launchdarkly/ldcli/assets/55991524/a898fd38-d1a9-496f-bdae-db557a032cbf)